playbooks/roles/forklift/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: 'Check variables defined'
+  fail:
+    msg: 'Please define forklift_name which deteremines the file boxes are deployed to'
+  when: forklift_name is undefined
+
+- name: 'Check Forklift state'
+  fail:
+    msg: 'The forklift_state variable is undefined. Please use pass "-e "forklift_state=up"" to spin up the boxes or "-e "forklift_state=destroy"" to destroy the boxes'
+  when: forklift_state is undefined
+
+- name: 'Ensure .tmp_boxes directory'
+  file:
+    state: directory
+    path: "{{ lookup('pipe','dirname `pwd`') }}/.tmp_boxes"
```

Comment:
Is this safe to assume? Given this logic is duplicated I also think it'd nicer to capture it into a variable.